internal/build/job/maker.go

```diff
@@ -113,14 +113,23 @@ func (m *maker) MakeJob(mod ootov1alpha1.Module, buildConfig *ootov1alpha1.Build
 	return job, nil
 }
 
+//FIXME:ybettan: if no one else uses it we should update it to a single object instead of a list
 func MakeSecretVolumes(secretRefs []v1.LocalObjectReference) []v1.Volume {
 	volumes := make([]v1.Volume, 0, len(secretRefs))
 
 	for _, secretRef := range secretRefs {
 		vol := v1.Volume{
 			Name: volumeNameFromSecretRef(secretRef),
 			VolumeSource: v1.VolumeSource{
-				Secret: &v1.SecretVolumeSource{SecretName: secretRef.Name},
+				Secret: &v1.SecretVolumeSource{
+					SecretName: secretRef.Name,
+					Items: []v1.KeyToPath{
+						{
+							Key:  ".dockerconfigjson",
```

Comment:
This function was designed to build a list of volumes referencing build secrets. This breaks an existing feature.

Reply:
Fixed.